si_bandit/cli.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
```

Comment:
All the components live in `thoth` namespace - `thoth/si_bandit`.

I guess you moved this to `si_bandit` to make sure bots can operate on versioning correctly. :+1: I'm wondering if we could extend kebechet to operate on `app.py` as well. In that case all the python code would live in one `app.py` file and we would have closer integration with openshift python s2i.

Reply:
I'm not sure what you are saying. Do we want to get rid of `cli.py`

Reply:
My point was to extend Kebechet's version manager so that it integrates with OpenShift conventions. If you run a simple OpenShift application that has solely just `app.py` file, we can still manage its version in Thoth.

This might be easier for very small code snippets that we need to run in the cluster so we do not enforce too much when us/users want to integrate with Thoth.

I'll open a PR.

Reply:
https://github.com/thoth-station/kebechet/pull/411